css/styles.css

```diff
@@ -1,3 +1,58 @@
-body {
-  display: block;
+.container {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.title {
+  text-align: center;
+}
+
+.book-author-collections {
+  border: 1px solid #333;
+}
+
+.book-author--card {
+  margin: 0 0.5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.form-wrapper {
+  display: flex;
+  justify-content: center;
+  padding: 2em;
+}
+
+form {
+  width: 60%;
+  padding: 1em;
+}
+
+.form-field {
+  margin: 0.5em 0;
+}
+
+.form-field input {
+  width: calc(100% - 0.5em);
+  height: 35px;
+}
+
+.btn-wrapper {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.btn {
+  width: fit-content;
+  min-width: 100px;
+  padding: 0.8em;
+}
+
+.btn:hover {
+  cursor: pointer;
+}
+
+.btn-remove {
+  padding: 0.4em;
```

Comment:
- [ ] Please make sure that your app style match [this wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) . We can see some differences between your design and the wireframe. Please fix it. As developer it is very important to pay attention to details  :smile: 

### Your design
![Screenshot from 2022-11-30 10-26-07](https://user-images.githubusercontent.com/6615394/204772362-4d5dc6f3-f042-4b98-8c17-12ac6ea12348.png)

### Wireframe
<img width="437" alt="awesome_books_core_elements" src="https://user-images.githubusercontent.com/6615394/204772368-8f58c19d-66a5-4cbd-ab23-d9bfe0002d25.png">